src/sentry/static/sentry/app/views/settings/projectSecurityHeaders/csp.jsx

```diff
@@ -20,16 +21,7 @@ const TextBlockNoMargin = styled(TextBlock)`
   margin-bottom: 0;
 `;
 
-class ProjectCspReports extends AsyncView {
-  static propTypes = {
-    setProjectNavSection: PropTypes.func,
-  };
-
-  componentWillMount() {
-    super.componentWillMount();
-    this.props.setProjectNavSection('settings');
```

Comment:
This might be needed for old project settings, not sure

Reply:
ya I might as well keep it since I kept it elsewhere